### PR TITLE
Add Dynamic Custom Otel Envars

### LINKED ISSUE #27: Add envar options for custom Otel config.
Add a way to add environment variables to custom otel config. Potentially using a dynamic method if possible.

### PR DESCRIPTION
**Pull request by Keimille**
- adds a way to dynamically add envars to task definition of the custom otel (values must be updated in console or with cli)
- adds variable `custom_otel_secrets` which is a type `list(string)` example: `custom_otel_secrets = ["var1", "var2", "etc"]`
- closes #27

### CODE REVIEW

File: main.tf
Comment (Mykolb) [inline main.tf:1]: I learned something new today.

**Review by Mykolb — APPROVED** [review]
LGTM

**Review by brandonjohnsonmediavine — APPROVED** [review]
Reviewed together during Code Review time. LGTM

**Comment by Keimille** [comment]
Just noting that we will not be using Spacelift registry for the time being.

**Pull request merged**